app/src/main/res/values/strings.xml

```diff
@@ -255,7 +255,9 @@
     <string name="onboarding_home_screen_section_useful_history_description_2">Revisit your latest searches from your homepage and tabs.</string>
 
     <!-- Onboarding home screen popup dialog, shown on top of the Jump back in section. Firefox is intentionally hardcoded. -->
-    <string name="onboarding_home_screen_jump_back_contextual_hint" tools:ignore="UnusedResources">Your personalized Firefox homepage now makes it easier to pick up where you left off. Find your recent tabs, bookmarks, and search results.</string>
+    <string name="onboarding_home_screen_jump_back_contextual_hint" moz:RemovedIn="105" tools:ignore="UnusedResources">Your personalized Firefox homepage now makes it easier to pick up where you left off. Find your recent tabs, bookmarks, and search results.</string>
+    <!-- Onboarding home screen popup dialog, shown on top of the Jump back in section. Firefox is intentionally hardcoded. -->
```

Comment:
was this comment duplicated?

Reply:
I removed the `Firefox is intentionally hardcoded.`

Reply:
Sorry, I forgot we duplicate the comment for a `_2` string